ci/jenkinsfiles/release.groovy

```diff
@@ -0,0 +1,335 @@
+                pipelineLib.gitTag("${TAG}", "${MESSAGE}")
+              }
+            }
+          }
+        }
+        stage('Publish Coldstorage Package') {
+          steps {
+            container('maven') {
+              script {
+                echo """
+                  -------------------------------------------------
+                  Upload Coldstorage Package ${VERSION} to ${CONNECT_PROD_URL}
+                  -------------------------------------------------
+                """
+                if (DRY_RUN_RELEASE == 'false') {
+                  pipelineLib.uploadPackage("${VERSION}", 'connect-preprod', "${CONNECT_PROD_URL}")
+                }
+              }
+            }
+          }
+          post {
+            always {
+              archiveArtifacts (
+                allowEmptyArchive: true,
+                artifacts: 'nuxeo-coldstorage-package/target/nuxeo-coldstorage-package-*.zip'
+              )
+            }
+          }
+        }
+        stage('Git Push') {
+          steps {
+            container('maven') {
+              echo """
+                --------------------------
+                Git push ${TAG}
+                --------------------------
+              """
+              script {
+                if (DRY_RUN_RELEASE == 'false') {
+                  pipelineLib.gitPush("${TAG}")
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    stage('Release and Bump reference branch') {
+      when {
+        allOf {
+          not {
+            environment name: 'DRY_RUN', value: 'true'
+          }
+        }
+      }
+      steps {
+        container('maven') {
+          script {
+            //cleanup files updated by other stages
+            sh 'git reset --hard'
+            // increment minor version
+            String nextVersion =
+              sh(returnStdout: true, script: "perl -pe 's/\\b(\\d+)(?=\\D*\$)/\$1+1/e' <<< ${CURRENT_VERSION}").trim()
+            echo """
+              -----------------------------------------------
+              Update ${REFERENCE_BRANCH} version from ${CURRENT_VERSION} to ${nextVersion}
+              -----------------------------------------------
+            """
+            pipelineLib.gitCheckout("${REFERENCE_BRANCH}")
+            pipelineLib.updateVersion("${nextVersion}")
+            pipelineLib.gitCommit("Release ${VERSION}, update ${CURRENT_VERSION} to ${nextVersion}", '-a')
+            if (DRY_RUN_RELEASE == 'false') {
+              pipelineLib.gitPush("${REFERENCE_BRANCH}")
+            }
+          }
+        }
+      }
+    }
+  }
+  post {
+    always {
+      script {
+        currentBuild.description = "Release ${VERSION} from build ${RC_VERSION}"
+      }
+    }
+    success {
+      script {
+        // update Slack Channel
+        String message = "Successfully released ${VERSION} from build ${env.RC_VERSION}: ${BUILD_URL} :tada:"
```

Comment:
what is the meaning of `:tada:` :)

Reply:
Hello,
for example 
<img width="986" alt="Screenshot 2020-12-15 at 11 49 53" src="https://user-images.githubusercontent.com/59877775/102205592-ae63ee80-3ecb-11eb-851a-9a4f46e1f71b.png">